First/dailyExercises/BubbleSort.java

```diff
@@ -0,0 +1,31 @@
+package firstPackage.First.dailyExercises;
+
+public class BubbleSort {
+    public static void bubbleSort(int[] array) {
```

Comment:
Add space between class and method as per guideline.

Reply:
Done